includes/wf_crm_webform_base.inc

```diff
@@ -859,10 +861,10 @@ abstract class wf_crm_webform_base {
    * @return int|null Civi file id
    */
   public static function saveDrupalFileToCivi($id) {
-    $file = file_load($id);
+    $file = File::load($id);
     if ($file) {
       $config = CRM_Core_Config::singleton();
-      $path = file_unmanaged_copy($file->getFileUri(), $config->customFileUploadDir);
+      $path = FileSystemInterface::copy($file->getFileUri(), $config->customFileUploadDir);
```

Comment:
Drupal - 8.8.6, PHP 7.3.7. With the above line, i see an error like -

>Error: Non-static method Drupal\Core\File\FileSystemInterface::copy() cannot be called statically in wf_crm_webform_base::saveDrupalFileToCivi() (line 867 of /Applications/MAMP/htdocs/cividrupal/web/modules/contrib/webform_civicrm/includes/wf_crm_webform_base.inc)

on a pretty complex webform involving activities with file input. Should we call this statically? From https://www.drupal.org/node/3006851, I see `file_unmanaged_copy()` is replaced by `\Drupal::service('file_system')->copy()`. Can you pls confirm?

Reply:
Ah ok -> this is why I replaced it with ->` FileSystemInterface::copy `-> 

<img width="1180" alt="image" src="https://user-images.githubusercontent.com/5340555/83313273-be87db00-a1d2-11ea-930c-649353709b95.png">

Link to the entire report: https://dev.acquia.com/drupal9/deprecation_status/errors?name=webform_civicrm%20%2A

Would: `\Drupal::FileSystemInterface::copy` work?

Reply:
OK seeing that too -> 
![image](https://user-images.githubusercontent.com/5340555/83313505-b0868a00-a1d3-11ea-96e6-ce597b2269c8.png)

Reply:
Does this -> 
`$path = \Drupal::service('file_system')->copy($file->getFileUri(), $config->customFileUploadDir);`
 work for you?

Reply:
yes, that works without any problem 👍

Reply:
Excellent - I just updated that!